### PR TITLE
1157: Fix promote to server

### PR DESCRIPTION
**Pull request by f1sh1918**
### Short description

node & npm has to be installed before app-toolbelt

### CODE REVIEW

File: .circleci/src/jobs/promote_to_server.yml [hunk 1]
```diff
@@ -3,12 +3,12 @@ docker:
 steps:
   - checkout:
       path: ~/project
-  - install_app_toolbelt
   - run:
       name: Install Curl
       command: |
         sudo apt update
         sudo apt install curl -y
+  - install_app_toolbelt
```
Comment (michael-markl) [inline .circleci/src/jobs/promote_to_server.yml:11]: Npm / node is not installed in the previous step, only curl

Comment (f1sh1918) [reply .circleci/src/jobs/promote_to_server.yml:11]: monday morning... :D

Comment (f1sh1918) [reply .circleci/src/jobs/promote_to_server.yml]: updated

File: .circleci/config.yml [hunk 2]
```diff
@@ -954,6 +954,7 @@ jobs:
         steps:
             - checkout:
                 path: ~/project
+            - run: apt update -y && apt install -y nodejs npm
```
[the remark this answers is not in the document]
Comment (michael-markl) [reply .circleci/config.yml:957]: @f1sh1918 I think you reverted it again by accident

[the remark this answers is not in the document]
Comment (f1sh1918) [reply .circleci/config.yml:957]: yes i accidently accepted your suggestion on a generated file. Gonna revert both last commits

**Review by michael-markl — APPROVED** [review]

**Pull request merged**